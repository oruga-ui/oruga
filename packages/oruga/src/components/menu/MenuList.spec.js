import { mount } from '@vue/test-utils'
import OMenuList from './MenuList'

let wrapper

describe('OMenuList', () => {
    beforeEach(() => {
        const Menu = {
            components: { OMenuList },
            template: `
            <div>
                <o-menu-list label="Test">
                </o-menu-list>
            </div>
            `
        }
        wrapper = mount(Menu)
    })

    it('is called', () => {
        expect(wrapper.findAll('.o-menu-label').at(0).text()).toBe('Test')
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })

    it('slot label', () => {
        expect(wrapper.findAll('.o-menu-label').at(0).text()).toBe('Test')
    })
})
