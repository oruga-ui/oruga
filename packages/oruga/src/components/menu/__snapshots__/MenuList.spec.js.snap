// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OMenuList render correctly 1`] = `
<div>
    <div>
        <div class="o-menu-label">
            <!----> <span>Test</span>
        </div>
        <ul class="o-menu-list"></ul>
    </div>
</div>
`;
